config/config.go

```diff
@@ -67,12 +67,13 @@ const (
 	"daemon_end_point": "127.0.0.1:8080",
 	"daemon_group_name":"default_group",
 	"daemon_type": "grpc",
+    "free_call_enabled": false,
```

Comment:
Looks like this is not used anymore.

Reply:
Yes, we identify if free calls are enabled based on the metadata